### PR TITLE
Docs: add missing plugin roles, actions and scope

### PR DESCRIPTION
**Pull request by IevaVasiljeva**
**What is this feature?**

Adding documentation for plugin roles, actions and scope that we had forgot to document.

### CODE REVIEW

File: docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md [hunk 1]
```diff
@@ -159,6 +161,7 @@ The following list contains role-based access control scopes.
 | `orgs:*` <br> `orgs:id:*`                       | Restrict an action to a set of organizations. For example, `orgs:*` matches any organization and `orgs:id:1` matches the organization whose ID is `1`.                                                                                             |
 | `permissions:type:delegate`                     | The scope is only applicable for roles associated with the Access Control itself and indicates that you can delegate your permissions only, or a subset of it, by creating a new role or making an assignment.                                     |
 | `permissions:type:escalate`                     | The scope is required to trigger the reset of basic roles permissions. It indicates that users might acquire additional permissions they did not previously have.                                                                                  |
+| `plugins:*` <br> `plugins:id:*`                 | Restrict an action to a set of application plugins. Fpr example, `plugins:id:grafana-oncall-app` matches Grafana OnCall plugin, and `plugins:*` matches all application plugins.                                                                   |
```
Comment (alyssawada) [inline docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md:164]:
```suggestion
| `plugins:*` <br> `plugins:id:*`                 | Restrict an action to a set of application plugins. For example, `plugins:id:grafana-oncall-app` matches Grafana OnCall plugin, and `plugins:*` matches all application plugins.                                                                   |
```

**Review by alyssawada — COMMENTED** [review]
Just one nit typo, otherwise looks good to me!

**Review by vtorosyan — APPROVED** [review]
Looks good. 

One question somehow related is if `plugins:install` is relevant only to application plugins or works with any plugin? 

I am wondering if the wording might confuse readers or not.

**Comment by IevaVasiljeva** [comment]
> Looks good.
> 
> One question somehow related is if `plugins:install` is relevant only to application plugins or works with any plugin?
> 
> I am wondering if the wording might confuse readers or not.

Good question. I've also double-checked, and the role to install plugins is currently still behind a feature flag. So perhaps we don't want to document it at all yet.

Let's see what @gamab says when he gets back :)

File: docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md [hunk 2]
```diff
@@ -95,6 +95,8 @@ The following list contains role-based access control actions.
...
 | `orgs:write`                         | `orgs:*` <br> `orgs:id:*`                                                               | Update one or more organizations.                                                                                                                                                                |
 | `plugins.app:access`                 | `plugins:*` <br> `plugins:id:*`                                                         | Access one or more application plugins (still enforcing the organization role)                                                                                                                   |
+| `plugins:install`                    | n/a                                                                                     | Install and uninstall application plugins.                                                                                                                                                       |
```
Comment (gamab) [inline docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md:98]:
```suggestion
| `plugins:install`                    | n/a                                                                                     | Install and uninstall plugins.                                                                                                                                                       |
```

File: docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md [hunk 3]
```diff
@@ -95,6 +95,8 @@ The following list contains role-based access control actions.
...
+| `plugins:write`                      | `plugins:*` <br> `plugins:id:*`                                                         | Edit settings for one or more application plugins.                                                                                                                                               |
```
Comment (gamab) [inline docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md:99]:
```suggestion
| `plugins:write`                      | `plugins:*` <br> `plugins:id:*`                                                         | Edit settings for one or more plugins.                                                                                                                                               |
```

File: docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md [hunk 4]
```diff
@@ -159,6 +161,7 @@ The following list contains role-based access control scopes.
 | `orgs:*` <br> `orgs:id:*`                       | Restrict an action to a set of organizations. For example, `orgs:*` matches any organization and `orgs:id:1` matches the organization whose ID is `1`.                                                                                             |
 | `permissions:type:delegate`                     | The scope is only applicable for roles associated with the Access Control itself and indicates that you can delegate your permissions only, or a subset of it, by creating a new role or making an assignment.                                     |
 | `permissions:type:escalate`                     | The scope is required to trigger the reset of basic roles permissions. It indicates that users might acquire additional permissions they did not previously have.                                                                                  |
+| `plugins:*` <br> `plugins:id:*`                 | Restrict an action to a set of application plugins. For example, `plugins:id:grafana-oncall-app` matches Grafana OnCall plugin, and `plugins:*` matches all application plugins.                                                                   |
```
Comment (gamab) [inline docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md:164]:
```suggestion
| `plugins:*` <br> `plugins:id:*`                 | Restrict an action to a set of plugins. For example, `plugins:id:grafana-oncall-app` matches Grafana OnCall plugin, and `plugins:*` matches all plugins.                                                                   |
```

**Review by gamab — APPROVED** [review]
Great job thanks :)
As pointed out by @vtorosyan these apply to plugins in general.
Btw, I hadn't documented these roles at first because of https://github.com/grafana/grafana/pull/54845 and https://github.com/grafana/grafana/pull/55613
But this we paused these for now, I guess it makes sense to merge this PR.

**Pull request merged**